index.html

```diff
@@ -97,15 +97,15 @@ <h2>Multi-Post Stories</h2>
           </ul>
           <button type="button" class="action-buttons">See Project</button>
```

Comment:
- [ ] Kindly this button doesn't pass with the hover case, kindly make the color darker to pass.

![image](https://user-images.githubusercontent.com/35535130/160601443-00ac9606-5474-4866-b7ae-567ffad74af9.png)